docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -47,7 +45,7 @@ Here is an example of what that looks like using version {props.fuelCoreVersion}
 To start the node with a custom configuration, you can use the command below:
 
 ```sh
-fuel-core run --chain ./chainConfig.json --db-type in-memory --debug
+fuel-core run --snapshot ./your/path/to/chain_config_folder --db-type in-memory --debug
 ```
 
 To find an example chain configuration file for a specific `fuel-core` version, refer to the [`fuel-core`](https://github.com/FuelLabs/fuel-core/tree/v0.22.0/deployment/scripts/chainspec) repo.
```

Comment:
I removed this temporarily, but can you add back the code import [here](https://docs.fuel.network/guides/running-a-node/running-a-local-node/#chain-configuration) to show all of the config files?